osu.Server.Spectator/Hubs/EntityStore.cs

```diff
@@ -62,6 +70,7 @@ public async Task<ItemUsage<T>> GetForUse(long id, bool createOnMissing = false)
                     throw new KeyNotFoundException($"Attempted to get untracked entity {typeof(T)} id {id}");
```

Comment:
this unlikely branch could probably also use an increment if we're tracking missing keys, similar/same as line 48